### DISCUSSION TITLE
Windows implementation

### DISCUSSION THREAD

**Discussion by multiwirth** [kmicki/SteamDeckGyroDSU#23] [Ideas]
any chance for an version that works with windows?

**Comment by HWiwoiiii** [comment]
really need windows version pleasepleaseplease

**Comment by kmicki** [comment]
Sorry. Most things would need a rewrite. I'm too busy currently but if anybody else is willing to give it a go then I'll try to support.

**Comment by FireGate13** [comment]
please make this for windows..
